source/our-stacks/android/index.md

```diff
@@ -1,6 +1,63 @@
 ---
 title: Android
-date: 2016-09-29 15:27:11
+date: 2016-10-06 17:37:40
 layout: stack
 ---
-Total Android
\ No newline at end of file
+
+Android developers at Nodes strive to build awesome apps using the Material Design Guidelines and Google provided design tools. 
+
+Custom designs are important to Nodes and helping client get the best possible Android experience are high priority for the Android devs.
+
+### Tasks
+- Maintain internal frameworks  
+- Take over code bases and adapt/update to newest patterns
+- Implement and tweak iOS designs to the _true Android way_
+- Help the UX and UI team with the ever-changing design language on Android
+- Estimate changes and feature requests on ongoing projects
+- Refactor common used code into libraries/utilities
+- Work closely together with the QA team
+
+## Stack
+
+### Language
+
+##### Java
+Java 1.0 was released in 1995 and is still being used widely today - in part because of Android. Being an old language the toolchain is mature and features like annotation processing, instant-run and an excellent Java SDK provides a nice tool set for development.
+
+### IDE
+Android Studio is the natural choice on the platform with all the bells and whistles it brings. 
+
+### Android Studio plugins
+- [android-butterknife-zelezny](https://github.com/avast/android-butterknife-zelezny) - Generate view bindings from the xml file directly
+- *Markdown support* - Preview the markdown layout directly in AS
```

Comment:
Missing link to the plugin? Would be nice to be consistent and link to all of them, if we can.